webknossos/tests/test_annotation.py

```diff
@@ -36,6 +36,58 @@ def test_annotation_from_file() -> None:
         assert len(skeleton_lines) == 32
 
 
+@pytest.mark.vcr()
+def test_annotation_from_file_with_multi_volume() -> None:
+    annotation = wk.Annotation(
+        TESTDATA_DIR / "annotations" / "multi_volume_example_CREMI.zip"
+    )
+
+    volume_names = sorted(list(annotation.get_volume_layer_names()))
```

Comment:
`sorted` can directly consume the iterable:
```suggestion
    volume_names = sorted(annotation.get_volume_layer_names())
```